common/src/main/java/tachyon/Constants.java

```diff
@@ -197,6 +197,7 @@
   public static final String WORKER_PRINCIPAL_KEY = "tachyon.worker.principal";
   public static final String WORKER_USER_TEMP_RELATIVE_FOLDER = "users";
   public static final String WORKER_DATA_SERVER = "tachyon.worker.data.server.class";
+  public static final String WORKER_COLLECT_METRICS = "tachyon.worker.collect.metrics";
```

Comment:
This is kind of misleading since it doesn't turn off the worker's own metrics collection?